### PR TITLE
[move 2024 migration guide] Mention rebuilding IDE

### PR DESCRIPTION
**Pull request by tnowacki**
## Description 

- Added a section to the migration guide for IDE support 

## Test Plan 

- 👁️ 

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: docs/content/guides/developer/advanced/move-2024-migration.mdx [hunk 1]
```diff
@@ -26,12 +26,22 @@ To migrate a project to Move 2024 Beta:
...
-## Automatic migration {#automatic-migration}
+### Automatic migration {#automatic-migration}
 
 Move 2024 includes an automatic migration script that you can use by calling `sui move migrate` in the root of your Move project. Upon running, your console prompts you for which Move edition to use. If you select `2024.beta`, the script invokes the compiler and attempts to automatically update your code to avoid the [breaking changes](#breaking-changes) the update introduces (including marking structs as `public`, mutable variables with the `mut` keyword, avoiding restricted keywords, swapping `friend`s for `public(package)`, and even updating paths to global paths in many cases).
 
 After this is done, your console displays a diff of the changes the script intends to make. If you accept the changes, the script updates your code and your `Move.toml` file automatically. You are now using Move 2024 Beta.
 
+### Update IDE support
+
+If you are using the `move-analyzer`, be sure to rebuild and reinstall to get 2024 support:
```
Comment (awelc) [inline docs/content/guides/developer/advanced/move-2024-migration.mdx:37]:
```suggestion
If you are using the [move-analyzer VSCode extension](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer), be sure to rebuild and reinstall the `move-analyzer` binary to get 2024 support:
```

**Review by awelc — CHANGES_REQUESTED** [review]
We should make sure that folks do not confuse this with `sui-move-analyzer`

**Comment by tnowacki** [comment]
> We should make sure that folks do not confuse this with `sui-move-analyzer`

Isn't that what your suggestion did?

**Comment by awelc** [comment]
> > We should make sure that folks do not confuse this with `sui-move-analyzer`
> 
> Isn't that what your suggestion did?

That was the intention. Folks can now see from the link which VSCode extension in particular we have in mind, and will not expect that this is, for example, intended to work with MoveBit's VSCode extension.

**Comment by tnowacki** [comment]
> > > We should make sure that folks do not confuse this with `sui-move-analyzer`
> > 
> > 
> > Isn't that what your suggestion did?
> 
> That was the intention. Folks can now see from the link which VSCode extension in particular we have in mind, and will not expect that this is, for example, intended to work with MoveBit's VSCode extension.

Then what's the change you want me to make? I'm confused

**Review by awelc — APPROVED** [review]
LGTM

**Pull request merged**